WordPress/WordPressApi/WordPressComOAuthClient.m

```diff
@@ -129,4 +129,16 @@ - (NSError *)processError:(NSError *)error forOperation:(AFHTTPRequestOperation
     return error;
 }
 
+- (id)cleanedUpResponseForLogging:(id)response {
+    if (![response isKindOfClass:[NSDictionary class]]) {
+        return response;
+    }
+    if (![(NSDictionary *)response objectForKey:@"access_token"]) {
```

Comment:
Small nitpick here you can ignore ;), but I think statements like this are easier/quicker to read when written like `statement != nil` versus `!statement`.